core/commands/commit_compare.py

```diff
@@ -5,22 +5,20 @@
 
 from ..git_command import GitCommand
 from ...common import util
-from ...common import ui
-from .log import GsLogBranchCommand
 
 
 COMMIT_NODE_CHAR = "●"
 COMMIT_NODE_CHAR_OPTIONS = "●*"
 COMMIT_LINE = re.compile("[%s][ /_\|\-.]*([a-z0-9]{3,})" % COMMIT_NODE_CHAR_OPTIONS)
 
 
-class GsCompareCommitCommand(TextCommand, GitCommand):
```

Comment:
I guess WindowCommand is more appropriate since we don't need anything from the active view.